tests/integration_tests.rs

```diff
@@ -415,6 +415,30 @@ fn pager_value_bat() {
         .failure();
 }
 
+#[test]
+fn pager_most() {
+    bat()
+        .env("PAGER", "most")
+        .arg("--paging=always")
+        .arg("test.txt")
+        .assert()
+        .success()
+        .stderr(predicate::eq("\x1b[33m[bat warning]\x1b[0m: Ignoring PAGER=\"most\": Coloring not supported. Override with BAT_PAGER=\"most\" or --pager \"most\"\n").normalize())
```

Comment:
maybe something like `predicate::contains("Ignoring PAGER")` or similar would be enough here? The warning text (and the escape sequences) are otherwise duplicated through the code base.

Reply:
We'll need to find some different way to test this now that we removed the warning. I left a TODO in the new code that I will fix as soon as we are all happy with the behavior itself.